test/argo-callbacks.ts

```diff
@@ -36,7 +36,7 @@ describe('Argo Callbacks', function () {
   // test progress is updated correctly
   describe('when the result handler sends a batch completion notice', function () {
     hookHttpBackendEach(function () {
-      return wmsRequest(this.frontend, collection, { ...validGetMapQuery, crs: 'ASYNC', layers: collection }).ok(() => true);
+      return wmsRequest(this.frontend, collection, { ...validGetMapQuery, crs: 'ASYNC', layers: collection, turbo: false }).ok(() => true);
```

Comment:
I think this is fine for this PR, but it's racking up a bit of debt, because ultimately we're going to have to fix these tests before removing Argo.  I assume what this is trying to do is force it to not use scrolling?